tests/add_fee_amount.go

```diff
@@ -0,0 +1,82 @@
+package tests
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ava-labs/subnet-evm/accounts/abi/bind"
+	teleportermessenger "github.com/ava-labs/teleporter/abi-bindings/go/Teleporter/TeleporterMessenger"
+	"github.com/ava-labs/teleporter/tests/network"
+	"github.com/ava-labs/teleporter/tests/utils"
+	"github.com/ethereum/go-ethereum/common"
+	. "github.com/onsi/gomega"
+)
+
+func AddFeeAmountGinkgo() {
+	addFeeAmount(&network.LocalNetwork{})
+}
+
+func addFeeAmount(network network.Network) {
```

Comment:
Let's export this function so that a separate application can call it to test against Fuji